harvard-lts/jstor-integration-tests
# put back certs for filebased, wont be used in k8s

**unsigned certs in supervisord-filesys.conf.**
* * *

**JIRA Ticket**: [(link)](https://at-harvard.atlassian.net/browse/LTSK8S-494)

# What does this Pull Request do?
Removing unsigned certs from supervisord-filesys.conf, which we toggle to for current (non-k8s) env caused harvester, transform, publisher components to time out when jenkins attempted to run itest, b/c curl to itest expected https. This pr reintroduces the unsigned certs into supervisord-filesys.conf for the current env, in k8s env we will toggle to supervisord.conf (no certs)

# How should this be tested?

Deploy this branch (checkout as a trial branch and push) and manually run trial for harvester in jenkins, the dev itest should pass.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
